src/scopes/trainingReports/index.test.js

```diff
@@ -119,6 +119,17 @@ describe('filtersToScopes', () => {
       expect(found.length).toBe(1);
       expect(found[0].id).toBe(gteEventReportPilot.id);
     });
+
+    it('returns an empty object when date range is invalid', async () => {
+      const filters = { 'startDate.win': '2021/06/07' };
+      const { trainingReport: scope } = await filtersToScopes(filters);
+      expect(scope).toEqual([{}]);
+    });
+
```

Comment:
`.toHaveBeenCalledWith({}, ['1', '2'], false, '~*')` isn't applicable here because it's not called with the comparator - so it correctly *isn't* called with that last argument, but this should satisfy the one-and-only uncovered branch in utils.js.